infrastructure/cdn-in-a-box/variables.env

```diff
@@ -89,11 +89,12 @@ TO_PERL_HOST=trafficops-perl
 TO_PERL_PORT=443
 TO_PERL_SCHEME=https
 TO_SECRET=blahblah
-TO_LOG_ERROR=stdout
-TO_LOG_WARNING=stdout
-TO_LOG_INFO=stdout
-TO_LOG_DEBUG=stdout
-TO_LOG_EVENT=stdout
+TO_LOG_ERROR=/var/log/traffic_ops/error.log
+TO_LOG_WARNING=/var/log/traffic_ops/warning.log
+TO_LOG_INFO=/var/log/traffic_ops/info.log
+#TO_LOG_DEBUG=/var/log/traffic_ops/debug.log
+TO_LOG_DEBUG=/dev/null
```

Comment:
`/dev/null` is intentional. Debug-level logs are excessively verbose and include the SQL for queries. I kept that `TO_LOG_DEBUG=/var/log/traffic_ops/debug.log` line there in case anyone *does* want to see the debug output.